doc/source/drivers/vector/csv.rst

```diff
@@ -537,6 +538,19 @@ Examples
        -oo Y_POSSIBLE_NAMES=latitude \
        -a_srs 'EPSG:4326'
 
+-  With both input and output CSV, use `ogr2ogr -segmentize`,
```

Comment:
```suggestion
-  Use `ogr2ogr -segmentize` to densify a input geometry being specified in the ``WKT`` special field. Note that one needs to specify the GEOMETRY=AS_WKT layer creation option, otherwise the input geometry would be returned unmodified:
```